.github/workflows/kind-e2e.yaml

```diff
@@ -49,14 +49,10 @@ jobs:
         restore-keys: |
           ${{ runner.os }}-go-
 
-    - name: Install Dependencies
-      run: |
-        set -x
-        echo "::group:: install ko ${KO_VERSION}"
-        curl -L https://github.com/google/ko/releases/download/v${KO_VERSION}/ko_${KO_VERSION}_linux_x86_64.tar.gz  | tar xzf - ko
-        chmod +x ./ko
-        sudo mv ko /usr/local/bin
-        echo "::endgroup::"
+    - name: Setup ko
+      uses: imjasonh/setup-ko@v0.4
+      with:
+        version: v${KO_VERSION}
```

Comment:
I used `KO_VERSION` (currently 0.11.2) but we can set `latest-release` - [ref](https://github.com/imjasonh/setup-ko/blob/cb157faaaf54568a909edf271e31e59f276716c1/action.yml#L27-L28).
I don't have strong opinion about the ko version so please let me know if any comment :smile:

Reply:
let's use latest-release